minkindr/include/kindr/minimal/implementation/rotation-quaternion-inl.h

```diff
@@ -407,8 +409,9 @@ RotationQuaternionTemplate<Scalar>::operator*(
 
   // check if the multiplication has resulted in the quaternion no longer being
   // approximately normalized.
-  if (std::abs(result.squaredNorm() - static_cast<Scalar>(1.0)) >
-      EPS<Scalar>::normalization_value()) {
```

Comment:
oh but this seems like an issue how angular velocity is integrated. if we do it properly with the exponential map it should be correct